test/companyAccountSpec.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const testCompanySchema = mongoose.Schema({
+    username: {type: String, required: true},
+    email: {type: String, required: true},
+    admin: {type: String},
+    company: {type: String},
+    picture: {type: Number, required: true},
+    bannerColor: {type: String},
+});
+
+const testCompanyAccount = mongoose.model('testCompany', testCompanySchema);
+describe('Connecting to database for company account', function() {
+  before(function (done) {
+    mongoose.connect('mongodb://soen341:soen341@soen341-shard-00-00-ruxjj.mongodb.net:27017,soen341-shard-00-01-ruxjj.mongodb.net:27017,soen341-shard-00-02-ruxjj.mongodb.net:27017/test?ssl=true&replicaSet=SOEN341-shard-0&authSource=admin');
+    const db = mongoose.connection;
+    db.on('error', console.error.bind(console, 'connection error'));
+    db.once('open', function() {
+      console.log('We are connected to test database for company account!');
+      done();

```

Comment:
Please keep all indentation to four spaces. (This applies to the rest of the file as well.)